warc_metadata_sidecar.py

```diff
@@ -112,6 +116,15 @@ def find_language(bytes_load):
         return None
 
 
+def determine_soft404(bytes_payload):
+    """Determine the probability of the record being a soft 404 record."""
+    detected = soft404.probability(bytes_payload.decode('utf-8', 'replace'))
+    if detected:
+        return detected
+    else:
+        return None
```

Comment:
I am wondering about the `if detected` and `else` returning `None`. It seems like if the probability returned is `0.0` it would `return None`, but I would think we would want to return that value, not `None`. For what case are you using the `if detected` to handle?

Reply:
I wasn't sure if there might be an instance where it doesn't return a decimal, but I don't think that's happening so I'll just return what we get from soft404.probability.